packages/modal/addon/styles/components/_nucleus-modal.scss

```diff
@@ -39,6 +39,10 @@ $slider-width: 600px;
     display: flex;
     flex-direction: column;
 
+    &--xlarge {
```

Comment:
Lets add to documentation too?

also we need to add a backstop test with this specification